### DISCUSSION TITLE
How can I use a slash at the end of a folder?

### DISCUSSION THREAD

**Discussion by slowcoder76** [datenstrom/community#775] [Ask a question] [answered]
Hello, `https://domain.com/my-site/` works only without the slash on the end on my web server.

Regards

Tip: You can edit your own questions here on GitHub, click on `...` -> `Edit`.

**Comment by IngoBelka** [comment, marked as answer]
This is less a problem of Yellow but one of settings in the .htaccess, which is situated in the root path of your `domain.com`. You have to set url rewritings there. [Apache mod_rewrite. ](https://httpd.apache.org/docs/current/mod/mod_rewrite.html )

**Reply by slowcoder76** [reply]
Thx
